libcextract/FunctionDepsFinder.hh

```diff
@@ -43,6 +43,33 @@ class ClosureSet
     Dependencies.insert(decl);
   }
 
+  /*
+   * Check if the same Decl was already added into the closure but for a
+   * different typename.
+   */
+  TypedefDecl *insideRangeOfDecl(TypedefDecl *decl)
```

Comment:
Please use `Decl` and not only `TypedefDecl`.

Reply:
There were some complaints from the compiler when I was using Decl, so I used the more specialized type. I'll check it tomorrow.